### PR TITLE
Fixing issue JAP-9

### PR DESCRIPTION
**Pull request by laurentschoelens**
Fixes https://github.com/highsource/jaxb2-annotate-plugin/issues/9
Unescape escaped strings annox gets and that get quoted after by codemodel.

### CODE REVIEW

**Review by mattrpav — CHANGES_REQUESTED** [review]

File: jaxb-annox-parent/jaxb-annox/src/main/java/org/jvnet/jaxb/annox/model/annotation/value/XStringAnnotationValue.java [hunk 1]
```diff
@@ -1,9 +1,13 @@
 package org.jvnet.jaxb.annox.model.annotation.value;
 
+import org.apache.commons.text.StringEscapeUtils;
```
[the remark this answers is not in the document]
Comment (mattrpav) [reply jaxb-annox-parent/jaxb-annox/src/main/java/org/jvnet/jaxb/annox/model/annotation/value/XStringAnnotationValue.java:3]: .. or we could merge this for v2.0.5 and make a new ticket about reducing commons-* dependencies.

[the remark this answers is not in the document]
Comment (laurentschoelens) [reply jaxb-annox-parent/jaxb-annox/src/main/java/org/jvnet/jaxb/annox/model/annotation/value/XStringAnnotationValue.java:3]: I tried to only unescape \\ in self hosted method but the problem also occurs for all quoted chars in string value.
I agree about not depending on external lib but this method in stringescapeutils is not simple so I decided to go with it.
We can do another ticket to remove this after in another release (commons-text is pretty new and replaces some commons-lang3 methods)

[the remark this answers is not in the document]
Comment (mattrpav) [reply jaxb-annox-parent/jaxb-annox/src/main/java/org/jvnet/jaxb/annox/model/annotation/value/XStringAnnotationValue.java:3]: Opened #333 to make a pass at removing dependencies where possible

**Review by mattrpav — APPROVED** [review]
LGTM

**Pull request merged**